package.json

```diff
@@ -144,6 +144,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.15.2",
     "extract-loader": "^5.0.1",
+    "fsevents": "^2.1.3",
```

Comment:
why is this added? Is it a peer dep of unionfs?

Reply:
nope is a relict of a further try will remove it :)

Had it before unionfs to mock the fs but unionfs is way easier to handle